test_cli_remapping/test/test_cli_remapping.py

```diff
@@ -0,0 +1,173 @@
+# Copyright 2018 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import asyncio
+import functools
+import os
+import random
+import sys
+import time
+
+from launch import LaunchDescriptor
+from launch.exit_handler import primary_exit_handler
+from launch.launcher import DefaultLauncher
+import pytest
+import rclpy
+
+
+def get_environment_variable(name):
+    """Get enironment variable or raise if it does not exist."""
+    path = os.getenv(name)
+    if not path:
+        raise EnvironmentError('Missing environment variable %r' % name)
```

Comment:
Nit: quoting the env var printed.
Also why do we need %r here ?

Reply:
`%r` isn't necessary. I used it to make the output have quotes around `name`. `"%s"` would work too.

Reply:
%r -> "%s"
327c0fd